.ci/update-upstream.sh

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env bash
```

Comment:
This script isn't used by the CI (right?) but is placed in the `.ci` folder. I'd rather move it to the `gradle` folder so that it is next to the version cataloge.

Reply:
Good point, I think we'll run it either manually to switch to a snapshot of whenever there is a new upstream release, so it's better to move it where you've suggested.